dist/adhese.min.js

```diff
@@ -1,3 +1 @@
-st.attach(e,"blur",B),Ut=de),at.replace({id:d,name:c,src:Yt.renderFile,_pos_id:i},l,s,J,z),Vt[i]=c)}}function se(e){var t=Vt[e];return t?ft({},t):null}function ae(){return Gt}var ce=null,de=!0,he=!1,le=6e4,pe="sf_pos",ue="sf_pos_rel_el",fe=750,ge=1,me=2,ve=10,ye="onBeforePosMsg",we="onPosMsg",be={"exp-ovr":1,"exp-push":0,bg:0,pin:0,"read-cookie":0,"write-cookie":0},Ae="exp-ovr",xe="collapse",ke="error",Se="geom-update",Te="focus-change",Ce=3e3,Ee="object",Pe="string",Ie="style",_e="length",Ne="width",Fe="height",He="PX",De="clip",Oe="scroll",je="onscroll",Ve="compatMode",Me="documentElement",Le="document",Re="nodeType",We="contains",Be="compareDocumentPosition",qe="elementFromPoint",Xe="auto",Ue="hidden",ze="overflow",$e="toFixed",Ge="attach",Je="detach",Qe="message",Ye="postMessage",Ke="guid",Ze="application/x-shockwave-flash",et=e&&e.$sf,tt=et&&et.ver,it=et&&et.env,rt=it&&it.ua,nt=et&&et.lib,ot=nt&&nt.lang,st=nt&&nt.dom,at=st&&st.iframes,ct=ot&&ot.cbool,dt=ot&&ot.cnum,ht=ot&&ot.cstr,lt=ot&&ot.callable,pt=ot&&ot.noop,ut=ot&&ot[Ke],ft=ot&&ot.mix,gt=st&&st.elt,mt=st&&st.par,vt=(st&&st.tags,st&&st.attr),yt=st&&st.doc,wt=st&&st.tagName,bt=st&&st.view,At=at&&at.view,xt=(st&&st.purge,st&&st.ready,e&&e.escape),kt=e&&e.Math,St=kt&&kt.max,Tt=kt&&kt.min,Ct=kt&&kt.round,Et=ce,Pt=ot&&ot.ParamHash,It=e&&e[Le],_t=it&&it.isIE,Nt=rt&&rt.ie||0,Ft=rt&&rt.webkit||0,Ht=rt&&rt.gecko||0,Dt=rt&&rt.opera||0,Ot=e&&e.location,jt=Ot
```

Comment:
shouldn't we build this file from something that is editable ?

Reply:
That's a build artifact that's checked into the source control. That's how we roll in this repo apparently :)